index.js

```diff
@@ -183,3 +183,28 @@ export function shareToTimeline(data) {
 export function shareToSession(data) {
   return nativeShareToSession(data);
 }
+
+/**
+ * wechat pay
+ * @param {Object} data
+ * @param {String} data.partnerId
+ * @param {String} data.prepayId
+ * @param {String} data.nonceStr
+ * @param {String} data.timeStamp
+ * @param {String} data.package
+ * @param {String} data.sign
+ * @returns {Promise}
+ */
+export function pay(data) {
+  return new Promise((resolve, reject) => {
+    WeChat.pay(data,() => {});
```

Comment:
This 2nd argument is not used correctly, propose:

```js
return new Promise((resolve, reject) => {
  WeChat.pay(data, () => resolve());
}).then(() => {
  emitter.on('PayReq.Resp', ...);
});
```

But I'm not sure if there is something problems when listening "PayReq.Resp" after the callback get invoked :)